e2e/_suites/fleet/fleet.go

```diff
@@ -1176,3 +1184,161 @@ func inputs(integration string) []kibana.Input {
 	}
 	return []kibana.Input{}
 }
+
+func metricsInputs(integration string, set string) []kibana.Input {
+	data := readJSONFile("./features/metrics.json", integration, set)
```

Comment:
I meant moving the JSON file for tests to a `testresources` directory. Could you move the file there? 🙏 

```suggestion
	data := readJSONFile("./testresources/metrics.json", integration, set)
```

Reply:
It would be great to also use `filepath` to honour current OS' path separator. There are several examples in the codebase for `filepath.Join`

Reply:
We defintely need this to handle differences between operating systems

Reply:
Done

Reply:
Done